src/manage_nsfs/manage_nsfs_constants.js

```diff
@@ -78,6 +79,8 @@ const OPTION_TYPE = {
     ips: 'string',
 };
 
+const ALLOW_BUCKET_CREATION_VALUES = ['true', 'false'];
```

Comment:
I suggest changing it to BOOLEAN_STRING_VALUES because we will soon support it on regenerate, show_secrets and wide